circle.yml

```diff
@@ -1,15 +1,12 @@
+machine:
+  node:
+    version: 4.4.7
 dependencies:
-  pre:
-    # Force updating wget due to the current containers being too out of date
-    - sudo apt-get update
-    - sudo apt-get install wget
   override:
-    - wget -O atom-amd64.deb https://atom.io/download/deb
-    # - sudo apt-get update # Cut out until wget is fixed on the containers
+    - curl -L https://atom.io/download/deb -o atom-amd64.deb
     - sudo dpkg --install atom-amd64.deb || true
-    - sudo apt-get -f install -y
-    - atom -v
-    - apm -v
+    - sudo apt-get update
+    - sudo apt-get -f install
     - apm install
```

Comment:
Before this line runs should be:

```yml
  - npm install
  - npm run lint
  - rm -Rf node_modules
```

Then change this line to `apm install --production`, and add a `npm install` to cover any devDependencies that specs might need but to avoid the warning from `eslint@3` (which you aren't seeing here for some reason).